### DISCUSSION TITLE
Scale by quantity

### DISCUSSION THREAD

**Discussion by ekcom** [mealie-recipes/mealie#2861] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Currently, there is no scale option to "one-and-a-half" a recipe.
Doing so requires me to do math on a calculator, which is not ideal.

Another example is if a recipe makes twelve (12) servings and I want sixteen (16) servings.
Mealie has no support for this, and back to the calculator I go.

### Please provide a concise description of the feature that would resolve your issue.

Scale by quantity: Changing the scale no longer *doubles* the portions but instead bumps it by one.
E.g., if a recipe makes twelve (12) servings, pressing the `+` to increase the services changes it to thirteen (13), scaling the ingredients accordingly.

Potentially include options to change by one portion at a time *and* a button to double the portions (so that it is easy to make large-scale and fine-tuned adjustments).

Other cooking apps/websites have this feature.

### Please consider and list out some caveats or tradeoffs made in your design decision

Costs:

- Dev time
  - Not very much though
- Potentially more cluttered UI if including *both* single serving adjustments and double serving adjustment buttons
- Scaling recipes up by, e.g., 1.2x will lead to some weird scaling for the ingredients (like 5/12 tbsp)

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by ekcom** [comment]
I see that this was already requested in #1161 now, but the currently implemented solution does not suite my needs.

**Comment by michael-genson** [comment]
Also related to https://github.com/mealie-recipes/mealie/issues/1390

**Comment by Kuchenpirat** [comment]
This has been implemented by the scaling refactor in #4298
